<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicSyncToolClass</class>
 <widget class="QMainWindow" name="MusicSyncToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>音乐同步工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="localHead">
      <item>
       <widget class="QLabel" name="localText">
        <property name="text">
         <string>本地文件夹：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="availableSpaceLocal">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchLocal">
        <property name="toolTip">
         <string/>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="placeholderText">
         <string>搜索</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="RemoteHead">
      <item>
       <widget class="QLabel" name="RemoteText">
        <property name="text">
         <string>设备文件夹：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="availableSpaceRemote">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchRemote">
        <property name="inputMask">
         <string/>
        </property>
        <property name="placeholderText">
         <string>搜索</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidgetLocal">
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>标题</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>艺术家</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>专辑</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>流派</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>年份</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>音轨号</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="tableWidgetRemote">
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>标题</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>艺术家</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>专辑</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>流派</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>年份</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>音轨号</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="LocalControl">
      <item>
       <widget class="QPushButton" name="copyToRemote">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>复制选中的音乐到设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshLocal">
        <property name="text">
         <string>刷新列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="favoriteOnlyLocal">
        <property name="text">
         <string>仅显示喜爱的音乐</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="pageLocal">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lastPageLocal">
        <property name="text">
         <string>上一页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextPageLocal">
        <property name="text">
         <string>下一页</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="copyToLocal">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>复制选中的音乐到本地</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshRemote">
        <property name="text">
         <string>刷新列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="favoriteOnlyRemote">
        <property name="text">
         <string>仅显示喜爱的音乐</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="pageRemote">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lastPageRemote">
        <property name="text">
         <string>上一页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextPageRemote">
        <property name="text">
         <string>下一页</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="playControl">
          <property name="text">
           <string>播放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="playSlider">
          <property name="maximum">
           <number>99</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="playProgress">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="nowPlaying">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="volumeLabel">
          <property name="text">
           <string>音量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="volumeSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>打开文件夹</string>
     </property>
     <addaction name="actionLocal"/>
     <addaction name="actionRemote"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="actionSettings"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>音乐</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>扫描重复音乐</string>
     </property>
     <addaction name="actionDupeLocal"/>
     <addaction name="actionDupeRemote"/>
    </widget>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>日志</string>
    </property>
    <addaction name="actionClean_log_files"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionLocal">
   <property name="text">
    <string>本地文件夹</string>
   </property>
  </action>
  <action name="actionRemote">
   <property name="text">
    <string>设备文件夹</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionDupeLocal">
   <property name="text">
    <string>本地</string>
   </property>
  </action>
  <action name="actionDupeRemote">
   <property name="text">
    <string>设备</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionClean_log_files">
   <property name="text">
    <string>清理日志文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/MusicSyncTool.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MusicSyncToolClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
